haroon-ali-dev/event-manager
# 37 setup cypress fixes

References #37.

**How To Test Cypress Installation**
- Make sure you have ran `npm i`.
- Run `npm run cypress:open`.
- Select option for E2E Tests.
- Click 'Start Testing'.
- Click on the only test spec to run the dummy test inside it.

**Completed Tasks**
- Installed Cypress.
- Setup script in `package.json` to open Cypress.
- Setup E2E tests config for Cypress.
- Fixed issue of input box being too small on mobile in modal for adding members to events.
- Replaced Auth0 loading message with spinner placed around the center of the screen.
- Added page titles.
- Added favicon.
